src/components/views/settings/tabs/user/SecurityUserSettingsTab.js

```diff
@@ -230,7 +230,7 @@ export default class SecurityUserSettingsTab extends React.Component {
                 onChange={this._updateBlacklistDevicesFlag}
             />;
         }
-
+        if(SettingsStore.getValue(UIFeature.AdvancedSettings)) return null;
```

Comment:
This looks odd with the AdvancedSettings = false, showing Cryptography information.

Reply:
No worries, I have changed this to "SecurityAllowSessionEdits". I could create another flag but not sure if u would like it.

Reply:
This change was done long time ago by me when I had issue with cached config. What I realised now is there is "ShowEncryptionSetting" flag which controls this info already, implemented by Joseph so I have completely removed that line and tested Security Setting. So **Encryption** is parent label, underneath comes cryptography so since encryption already hides all stuff there is no need to have this additional flag in commit above so I have removed it entirely and it works perfectly.